owlbot.py

```diff
@@ -32,7 +32,7 @@
 templated_files = common.py_library(
     unit_test_python_versions=["3.7", "3.8", "3.9", "3.10"],
     system_test_python_versions=["3.7", "3.8", "3.9", "3.10"],
-    cov_level=86,
+    cov_level=88,
```

Comment:
Any way to get rid of the magic threshold quickly? Probably deserves a separate PR?

Reply:
I've got https://github.com/googleapis/python-bigquery-pandas/issues/392 open to bump us up to 100%. 88% is a lot better than what it used to be...